ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -100,6 +105,12 @@ private[apiserver] final class ApiUserManagementService(
       } { case (user, pRights) =>
         for {
           _ <- identityProviderExistsOrError(user.identityProviderId)
+          authorizedUserContext <- authorizedUserContextFO
+          _ <- verifyPartyExistInIdp(
+            pRights,
+            user.identityProviderId,
+            authorizedUserContext.isParticipantAdmin,
+          )
           _ <- verifyUserIsNonExistentOrInIdp(user.identityProviderId, user.id)
```

Comment:
This is confusing: a user existing and being in the correct idp will result in a success from this method call.
Can you add a comment that this is nevertheless expected to result in an error from the user store layer?